plugins/modules/database/misc/odbc.py

```diff
@@ -31,7 +31,14 @@
         - Parameters to pass to the SQL query.
       type: list
       elements: str
-
+    commit:
+      description:
+        - Perform a commit after the execution of the SQL query.
+        - Some databases will allow a commit after a select whereas others will raise an exception.
```

Comment:
```suggestion
        - Some databases allow a commit after a select whereas others raise an exception.
```
this is a best practice to use a present tense instead of future in Ansible documentation whenever possible  (according to the dev guide)